scans_test.go

```diff
@@ -233,6 +234,50 @@ func createTestWatch(t *testing.T) (string, func()) {
 	}
 }
 
+// JAS docker scan tests
+
+func TestAdvancedSecurityDockerScan(t *testing.T) {
+	cleanup := initNativeDockerWithXrayTest(t)
+	defer cleanup()
+	runAdvancedSecurityDockerScan(t, "jfrog/demo-security:latest")
+}
+
+func runAdvancedSecurityDockerScan(t *testing.T, imageName string) {
+	// Pull image from docker repo
+	imageTag := path.Join(*securityTests.ContainerRegistry, securityTests.DockerVirtualRepo, imageName)
+	dockerPullCommand := container.NewPullCommand(containerUtils.DockerClient)
+	dockerPullCommand.SetCmdParams([]string{"pull", imageTag}).SetImageTag(imageTag).SetRepo(securityTests.DockerVirtualRepo).SetServerDetails(securityTests.XrDetails).SetBuildConfiguration(new(build.BuildConfiguration))
+	if assert.NoError(t, dockerPullCommand.Run()) {
+		defer commonTests.DeleteTestImage(t, imageTag, containerUtils.DockerClient)
+		args := []string{"docker", "scan", imageTag, "--server-id=default", "--format=simple-json", "--fail=false", "--min-severity=low", "--fixable-only"}
+
+		// Run docker scan on image
+		output := securityTests.PlatformCli.WithoutCredentials().RunCliCmdWithOutput(t, args...)
+		if assert.NotEmpty(t, output) {
+			verifyAdvancedSecurityScanResults(t, output)
+		}
+	}
+}
```

Comment:
You already have `runDockerScan` please modify it to fit your changes.
You can pass the verification method func to it...